src/App.js

```diff
@@ -1,25 +1,67 @@
 import React from 'react';
 
+import Clock from './components/Clock';
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    isClockVisible: 1,
+    clockName: 3.14,
+  };
 
+  switchClock = (buttonName) => {
+    this.setState({ isClockVisible: buttonName === 'Show Clock' ? 1 : false });
```

Comment:
I've forgotten that can write like : this.setState({ isClockVisible: buttonName === 'Show Clock' }); 🤦‍♂️

 I can't use your method because I call 1 method to two buttons